src/css/header.css

```diff
@@ -1,3 +1,124 @@
 .header {
-  background-color: lightcoral;
+  padding-bottom: 8px;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  background-color: var(--white);
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.logo {
+  font-family: var(--font-family);
+  font-weight: 700;
+  font-size: 20px;
+  text-transform: lowercase;
+  color: var(--black);
+}
+
+.logo-web {
+  color: var(--green);
+}
+
+.nav,
+.menu-soc-list {
+  display: none;
+}
+
+@media screen and (min-width: 1280px) {
+  .nav,
+  .menu-soc-list {
+    display: flex;
+  }
+}
+
+.nav-list {
+  gap: 20px;
+}
+
+.nav-link {

```

Comment:
Add `padding: 13px 0;` to make an clickable area bigger

Reply:
Done